backend/app/services/execution/polling_dispatcher.py

```
@@ -0,0 +1,352 @@
+                                    }
+                                    for item in summaries
+                                ]
+                        except json.JSONDecodeError:
+                            pass
+
+            elif gemini_event == "content.start" and index == 1:
+                return thought_summaries
+
+    except GeminiInteractionError:
+        pass
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)